src/ace.js

```diff
@@ -252,7 +252,8 @@ export default class ReactAce extends Component {
 
   onBlur(event) {
     if (this.props.onBlur) {
-      this.props.onBlur(event);
+      const value=this.editor.getValue();
```

Comment:
this would have significant impact on performance, would be better to pass this.editor, so that users can call `.getValue()` only when needed.

Reply:
@nightwing I will make that change the update the PR